wpilibc/src/main/native/include/frc/controller/PIDController.h

```diff
@@ -27,7 +27,7 @@ class PIDController : public frc::Sendable,
    * @param Ki     The integral coefficient.
    * @param Kd     The derivative coefficient.
    * @param period The period between controller updates in seconds. The
-   *               default is 20 milliseconds.
+   *               default is 20 milliseconds. Must be non-zero and positive.
```

Comment:
After some more discussion, we also want a reportError in the constructor if the period is 0 or less.